### PR TITLE
Implemented new progress HUD

### LINKED ISSUE #30: Use better indicators
**Actual Behaviour**

The indicator on loading should look nice or have some text.
Screenshot 2019-06-11 at 12 52 46 PM

**Expected Behaviour**

The `SVProgressHUD` Pod allows us to use a smooth and fine HUD, which I think looks and feels much better.
Screenshot 2019-06-11 at 12 51 20 PM

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by obitodarky**
- Fixes #30 
- Also fixed an issue where users were able to interact with the app while in the loading state

**Changes:** 

1. Added SVProgressHUD in Podfile
2. Removed old IndicatorView and used SVProgressHUD in the Welcome Screen.
3. Added a method to ignore Interaction Events while loading.

**Screenshots for the change:** 

Screenshot 2019-06-11 at 12 51 20 PM

### CODE REVIEW

**Comment by kushthedude** [comment]
@obitodarky Fix lint issues

**Review by kushthedude — CHANGES_REQUESTED** [review]
@obitodarky Fix lint issues

File: EventyayOrganizer/Controllers/WelcomeController/WelcomeVCMethods.swift [hunk 1]
```diff
@@ -65,9 +56,11 @@ extension WelcomeViewController {
         }
     }
     @IBAction func onGetStartedClicked(_ sender: Any) {
-        indicatorView.startAnimating()
+        SVProgressHUD.show(withStatus: "Logging In")
```
Comment (baquer) [inline EventyayOrganizer/Controllers/WelcomeController/WelcomeVCMethods.swift:59]: Put a string in a constant file.

File: Podfile.lock [hunk 2]
```diff
@@ -65,8 +68,9 @@ SPEC CHECKSUMS:
...
   NVActivityIndicatorView: 4ca19fccc84595a78957336a086d00a49be6ce61
   ObjectMapper: 0d4402610f4e468903ae64629eec4784531e5c51
+  SVProgressHUD: 1428aafac632c1f86f62aa4243ec12008d7a51d6
   SwiftLint: 7a0227733d786395817373b2d0ca799fd0093ff3
 
-PODFILE CHECKSUM: 365cb61113528df1a40b57d24476ce4432bbe17d
+PODFILE CHECKSUM: 662d237a6a69e6dd67394d62945c640aec8d1ae9
 
-COCOAPODS: 1.6.1
+COCOAPODS: 1.5.3
```
Comment (baquer) [inline Podfile.lock:76]: Update your pods to the latest version.

**Review by baquer — CHANGES_REQUESTED** [review]

**Comment by obitodarky** [comment]
I'll fix the lint issues and perform these changes and file a different PR. I am closing this for now.

**Pull request closed without merging**